README.md

```diff
@@ -80,4 +74,14 @@ ansible-vault decrypt '<path-to-file-to-decrypt>'
 
 ```bash
 ansible localhost -m debug -a var='<variable-name>' -e '@<path-to-file>' --ask-vault-pass
- ```
+```
+
+## Updating encrypted variables
+
+The `bootstrap.yml` playbook can be re-run to update variable values.
+
+1. Decrypt the `CREDENTIALS_CONFIG.yml` file as per [here](#decryption).
+
+2. Make the required update/s
+
+3. Re-run the `bootstrap.yml` playbook
```

Comment:
Similar to previous comment, would be nice to provide the exact bootstrap command:
```
ansible-playbook -i ./inventories/hosts bootstrap.yml --extra-vars='@CREDENTIALS_CONFIG.yml'
```